Src/CSharpier/SyntaxPrinter/FormattingContext.cs

```diff
@@ -1,3 +1,8 @@
 namespace CSharpier.SyntaxPrinter;
 
-public class FormattingContext { }
```

Comment:
Note: you should consider making this a record, since it's probably never going to have methods on it.